@here/harp-webpack-utils/package.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "@here/harp-webpack-utils",
+    "version": "0.11.1",
```

Comment:
0.1.0?

Reply:
To limit the git tags per release we try to keep the public packages in sync.